esphome/components/hdc1080/hdc1080.cpp

```diff
@@ -21,7 +21,9 @@ void HDC1080Component::setup() {
   };
 
   if (!this->write_bytes(HDC1080_CMD_CONFIGURATION, data, 2)) {
-    this->mark_failed();
+    // as instruction is same as powerup defaults (for now), interpret as warning if this fails
+    ESP_LOGW(TAG, "HDC1080 initial config instruction error");
+    //this->mark_failed();
```

Comment:
Let's make it an actual warning then:
```suggestion
    this->status_set_warning();
```
(Also, in general we don't like commented-out code, just remove it; if it's ever necessary there's always git).